gradle/libs.versions.toml

```diff
@@ -1,6 +1,10 @@
 [versions]
+# The java-compilation version is the latest supported (not necessarily LTS) version of Java. It should be identical to Java-version used in `actions/setup-java`
+java-compilation = "20"
+# The java-target version is the lowest supported LTS version of Java. Jar's produced are bytecode compatible with this version.
+java-target = "8"
 kotlin = "1.8.22"
-kotlinDev = "1.8.22"
+kotlinDev = "1.9.0"
```

Comment:
❗ 
To showcase the new setup now properly uses kotlinDev version, I updated `kotlinDev` version to 1.9.0. I hoped it also would be helpful to work on https://github.com/pinterest/ktlint/issues/2061, so I decided to keep it. 
Please let me know if I misunderstood its purpose, and it doesn't mean to be used as Kotlin version for local Kotlin development 👀